messenger.rst

```diff
@@ -1268,6 +1268,72 @@ during a request::
         :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`
         or :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\WebTestCase`.
 
+Amazon SQS
+~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The Amazon SQS transport has been added in Symfony 5.1
+    Install it by running:
+
+    .. code-block:: terminal
+
+        $ composer require symfony/amazon-sqs-messenger
+
+The ``SQS`` transport configuration looks like this:
+
+.. code-block:: bash
+
+    # .env
+    MESSENGER_TRANSPORT_DSN=sqs://AKIAIOSFODNN7EXAMPLE:j17M97ffSVoKI0briFoo9a@sqs.eu-west-3.amazonaws.com/messages
+    #MESSENGER_TRANSPORT_DSN=sqs://localhost:9494/messages?sslmode=disable
+
+.. note::
+
+    The transport will automatically create queues that are needed. This
+    can be disabled setting the "auto_setup" option to ``false``.
+
+A numbeXr of options can be configured via the DSN or via the ``options`` key
```

Comment:
`number`

Reply:
Ah.. I was testing if the docker build script automatically updated.. :)